packages/devtools_app/pubspec.yaml

```diff
@@ -0,0 +1,59 @@
+name: devtools_app
+description: Main package implementing web-based performance tooling for Dart and Flutter.
+
+# Note: when updating this version, please update the corresponding entry in
+# lib/devtools.dart and publish a new version of package:devtools as well.
```

Comment:
update comment to include both files that need to be updated.

Reply:
done.